sqlalchemy_bigquery/__init__.py

```diff
@@ -41,6 +41,9 @@
     BIGNUMERIC,
 )
 
+# Define a "dialect" attribute to follow the common SqlAlchemy convention assumed by some libraries that leverage SqlAlchemy.
```

Comment:
Other SqlAlchemy dialects define a `dialect` attribute, including but not limited to:
* [base.dialect = dialect = snowdialect.dialect](https://github.com/snowflakedb/snowflake-sqlalchemy/blob/38b8082f8ebf4106b6c9465996032b24806ad048/__init__.py#L62)
* [base.dialect = dialect = mysqldb.dialect](https://github.com/sqlalchemy/sqlalchemy/blob/76b506ed51e31b922014a30de2a5952d1a6ad891/lib/sqlalchemy/dialects/mysql/__init__.py#L60)
* [base.dialect = dialect = psycopg2.dialect](https://github.com/sqlalchemy/sqlalchemy/blob/76b506ed51e31b922014a30de2a5952d1a6ad891/lib/sqlalchemy/dialects/postgresql/__init__.py#L65)

See additional details on why this matters at #250.

Reply:
Interesting.  I wasn't aware of this.  Thanks!

This is a weird way to spell this. Why not define `dialect` in `base` and import it? :shrug: